SECURITY.md

```diff
@@ -2,10 +2,8 @@
 
 ## Supported Versions
 
-| Version | Supported          |
-| ------- | ------------------ |
-| 6.x     | :white_check_mark: |
-| <6.x    | :x:                |
+The supported versions of praw include the previous major release. 
```

Comment:
```suggestion
The supported versions of PRAW include the previous major release. 
```

Reply:
I'll have to figure out how to do that quickly.